src/store/flow/flow.ts

```diff
@@ -336,10 +336,10 @@ export const actions: ActionTree<IFlowsState, IRootState> = {
 
     commit('flow_addBlock', { block: duplicatedBlock })
 
-    router.replace({
-      name: 'block-selected-details',
-      params: { blockId: duplicatedBlock.uuid },
-    })
+    //router.replace({
```

Comment:
@poojahpatil90 , would the deep linking work without these ?

Reply:
I assume it wouldn't? Just trying to highlight where the problem is. We'll need a second PR for an actual fix I think?